# Slovak translation for apport
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-02-10 12:34+0100\n"
"PO-Revision-Date: 2019-11-10 11:57+0000\n"
"Last-Translator: Ľuboš Mudrák <lubosmudrak@azet.sk>\n"
"Language-Team: Slovak <sk@li.org>\n"
"Language: sk\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Získať informácie o systéme"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Pre získanie systémových informácií potrebných k ohláseniu tohto problému je "
"požadované overenie totožnosti"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Hlásenie o probléme so systémom"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""
"Zadajte prosím svoje heslo, aby mohol byť problém systémového programu "
"nahlásený."

#: ../apport/ui.py:166
msgid "This package does not seem to be installed correctly"
msgstr "Vyzerá to tak, že tento balík nie je nainštalovaný korektne"

#: ../apport/ui.py:176
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:206
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Máte nainštalovanú zastaralú verziu balíka. Prosím, aktualizujte nasledujúce "
"balíky a skontrolujte, či problém pretrváva:\n"
"\n"
"%s"

#: ../apport/ui.py:369
msgid "unknown program"
msgstr "neznámy program"

#: ../apport/ui.py:372
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Prepáčte, program \"%s\" neočakávane skončil"

#: ../apport/ui.py:375 ../apport/ui.py:1907
#, python-format
msgid "Problem in %s"
msgstr "Problém v %s"

#: ../apport/ui.py:380
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Váš počítač nemá dosť voľnej pamäte na automatickú analýzu problému a "
"poslanie hlásenia vývojárom."

#: ../apport/ui.py:440 ../apport/ui.py:458 ../apport/ui.py:631
#: ../apport/ui.py:640 ../apport/ui.py:877 ../apport/ui.py:1685
#: ../apport/ui.py:1868 ../apport/ui.py:1873
msgid "Invalid problem report"
msgstr "Neplatné hlásenie problému"

#: ../apport/ui.py:441
msgid "You are not allowed to access this problem report."
msgstr "Nemáte povolený prístup k tomuto hláseniu o probléme."

#: ../apport/ui.py:449
msgid "Error"
msgstr "Chyba"

#: ../apport/ui.py:451
msgid "There is not enough disk space available to process this report."
msgstr "Nie je dostatok voľného miesta na disku na spracovanie tohto hlásenia."

#: ../apport/ui.py:489
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:491
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:502 ../apport/ui.py:607
msgid "Invalid PID"
msgstr "Neplatné PID"

#: ../apport/ui.py:503
msgid "The specified process ID does not exist."
msgstr "Proces s zadaným identifikátorom neexistuje."

#: ../apport/ui.py:508
msgid "Not your PID"
msgstr ""

#: ../apport/ui.py:509
msgid "The specified process ID does not belong to you."
msgstr "Proces s zadaným identifikátorom vám nepatrí."

#: ../apport/ui.py:564
msgid "No package specified"
msgstr "Nebol špecifikovaný žiadny balík"

#: ../apport/ui.py:566
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Je potrebné špecifikovať balík alebo PID. Pozrite --help pre viac informácií."

#: ../apport/ui.py:594
msgid "Permission denied"
msgstr "Prístup odmietnutý"

#: ../apport/ui.py:596
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Nemáte dosah na označený proces. Prosím, spustite program ako jeho vlastník, "
"alebo ako root."

#: ../apport/ui.py:609
msgid "The specified process ID does not belong to a program."
msgstr "Zadaný identifikátor procesu nepatrí k programu."

#: ../apport/ui.py:633
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Skript %s nebol schopný určiť postihnutý balík"

#: ../apport/ui.py:641
#, python-format
msgid "Package %s does not exist"
msgstr "Balík %s neexistuje"

#: ../apport/ui.py:670 ../apport/ui.py:882 ../apport/ui.py:915
#: ../apport/ui.py:925
msgid "Cannot create report"
msgstr "Nie je možné vytvoriť hlásenie"

#: ../apport/ui.py:685 ../apport/ui.py:744 ../apport/ui.py:762
msgid "Updating problem report"
msgstr "Aktualizuje sa hlásenie o probléme"

#: ../apport/ui.py:687
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Nenahlásili ste toto hlásenie o chybe, alebo nie ste odberateľom noviniek o "
"tejto chybe, alebo je toto hlásenie duplikát alebo už bolo uzavreté.\n"
"\n"
"Prosím vytvorte nové hlásenie pomocou \"apport-bug\"."

#: ../apport/ui.py:699
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Nenahlásili ste toto hlásenie o chybe. Je oveľa jednoduchšie označiť chybu "
"ako duplikát inej chyby, ako preniesť vaše komentáre a prílohy do novej "
"chyby.\n"
"\n"
"Následovne odporúčame aby ste nahlásili novú chybu pomocou \"apport-bug\" a "
"pridali komentár k tejto chybe, o tej ktorú ste nahlásili.\n"
"\n"
"Naozaj chcete pokračovať?"

#: ../apport/ui.py:745 ../apport/ui.py:763
msgid "No additional information collected."
msgstr "Neboli zozbierané žiadne dodatočné informácie."

#: ../apport/ui.py:826
msgid "What kind of problem do you want to report?"
msgstr "Aký druh problému chcete nahlásiť?"

#: ../apport/ui.py:847
msgid "Unknown symptom"
msgstr "Neznámy príznak"

#: ../apport/ui.py:848
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Príznak \"%s\"  nie je známy."

#: ../apport/ui.py:884
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:900
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Pre nahlásenie problému s aplikáciou kliknite, prosím, po zavretí tejto "
"správy na okno aplikácie."

#: ../apport/ui.py:917 ../apport/ui.py:926
msgid "xprop failed to determine process ID of the window"
msgstr "xprop nedokázal zistiť identifikátor procesu okna"

#: ../apport/ui.py:941
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:942
msgid "Specify package name."
msgstr "Zadajte názov balíka."

#: ../apport/ui.py:949 ../apport/ui.py:1082
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Pridať k hláseniu značku navyše. Môže byť zadaná viackrát."

#: ../apport/ui.py:987
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:998
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Spustiť v režime nahlásenia chyby. Vyžaduje --package a voliteľný --pid, "
"alebo len --pid. Ak nie je zadaný ani jeden z nich, zobrazí sa zoznam "
"známych príznakov. (Predpokladané ak je zadaný len jeden argument.)"

#: ../apport/ui.py:1007
msgid "Click a window as a target for filing a problem report."
msgstr "Kliknite na okno ktoré bude cieľom pre vyplnenia hlásenia o chybe."

#: ../apport/ui.py:1016
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"Začnite v režime aktualizácie chyby. Môžete použiť voliteľný argument --"
"package."

#: ../apport/ui.py:1025
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Vytvorte chybové hlásenie o príznaku. (Predpokladané ak je zadané ako jediný "
"argument názov príznaku.)"

#: ../apport/ui.py:1034
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Zadajte názov balíka v režime --file-bug. Je to voliteľné ak je zadaný --"
"pid. (Predpokladané ak je zadané ako jediný argument názov.)"

#: ../apport/ui.py:1045
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Špecifikujte spustený program v režime --file-bug. S týmto upresnením bude "
"hlásenie obsahovať viac informácií. (Ak je číslo procesu uvedené ako jediný "
"argument.)"

#: ../apport/ui.py:1053
msgid "The provided pid is a hanging application."
msgstr "Poskytnutý pid je zamrznutá aplikácia."

#: ../apport/ui.py:1061
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Nahláste pád z daného .apport alebo .crash súboru namiesto zostávajúcich v "
"%s. (Predpokladané ak je zadaný ako jediný argument súbor.)"

#: ../apport/ui.py:1071
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"Uložiť hlásenie do súboru namiesto priameho odosielania. Tento súbor môže "
"byť nahlásený neskôr aj z iného zariadenia."

#: ../apport/ui.py:1090
msgid "Print the Apport version number."
msgstr "Zobraziť čislo verzie programu Apport."

#: ../apport/ui.py:1252
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr "Týmto spustíte apport-retrace v terminálovom okne na preskúmanie pádu."

#: ../apport/ui.py:1256
msgid "Run gdb session"
msgstr "Spustiť gdb reláciu"

#: ../apport/ui.py:1257
msgid "Run gdb session without downloading debug symbols"
msgstr "Spustiť gdb reláciu bez sťahovania ladiacich symbolov"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1259
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Aktualizovať %s pomocou úplného symbolického trasovania zásobníku"

#: ../apport/ui.py:1321
msgid "Can't remember send report status settings"
msgstr ""

#: ../apport/ui.py:1325
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""

#: ../apport/ui.py:1408 ../apport/ui.py:1421
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"Toto hlásenie problému sa vzťahuje na program, ktorý už nie je nainštalovaný."

#: ../apport/ui.py:1445
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr "Problém bol zaznamenaný v programe %s, ktorý bol po páde zmenený."

#: ../apport/ui.py:1506 ../apport/ui.py:1626 ../apport/ui.py:1877
msgid "This problem report is damaged and cannot be processed."
msgstr "Toto hlásenie problému je poškodené a nie je možné ho spracovať."

#: ../apport/ui.py:1515
msgid "This report is about a package that is not installed."
msgstr "Toto hlásenie je o balíku, ktorý nie je nainštalovaný."

#: ../apport/ui.py:1523
msgid "An error occurred while attempting to process this problem report:"
msgstr "Nastala chyba počas spracovania hlásenia o chybe:"

#: ../apport/ui.py:1540
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1545
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1546
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1583
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1588
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1687
msgid "Could not determine the package or source package name."
msgstr "Nedá sa zistiť balík alebo zdrojový názov balíka."

#: ../apport/ui.py:1712
msgid "Unable to start web browser"
msgstr "Nepodarilo sa spustiť internetový prehliadač"

#: ../apport/ui.py:1713
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Nepodarilo sa spustiť internetový prehliadač pre otvorenie %s."

#: ../apport/ui.py:1823
msgid "Network problem"
msgstr "Problém siete"

#: ../apport/ui.py:1827
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Nie je možné sa pripojiť do databázy pádov, prosím, skontrolujte vaše "
"internetové pripojenie."

#: ../apport/ui.py:1859
msgid "Memory exhaustion"
msgstr "Vyčerpanie pamäte"

#: ../apport/ui.py:1861
msgid "Your system does not have enough memory to process this crash report."
msgstr "Váš systém nemá dostatok pamäti na spracovanie chybového hlásenia."

#: ../apport/ui.py:1912
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Problém nemohol byť ohlásený:\n"
"\n"
"%s"

#: ../apport/ui.py:1968 ../apport/ui.py:1980
msgid "Problem already known"
msgstr "Problém je už známy"

#: ../apport/ui.py:1970
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Tento problém už bol nahlásený v chybovom hlásení zobrazenom v internetovom "
"prehliadači. Skontrolujte, prosím, či môžete doplniť ďalšie informácie, "
"ktoré môžu byť nápomocné pre vývojárov."

#: ../apport/ui.py:1982
msgid "This problem was already reported to developers. Thank you!"
msgstr "Tento problém bol už nahlásený vývojárom. Ďakujeme!"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "Pokračujte stlačením ľubovoľného klávesu..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "Čo by ste radi urobili? Vaše možnosti sú:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "Prosím, zvoľte (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i bajtov)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:405
msgid "(binary data)"
msgstr "(binárne dáta)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:198
msgid "Send problem report to the developers?"
msgstr "Odoslať hlásenie o chybe vývojárom?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Po odoslaní hlásenia o chybe, vyplňte, prosím, formulár v\n"
"automaticky otvorenom internetovom prehliadači."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "&Odoslať hlásenie (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "&Preskúmať lokálne"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "&Pozrieť hlásenie"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Uchovať súbor s hlásením pre neskoršie odoslanie alebo kopírovanie na iné "
"miesto"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Zrušiť a &ignorovať budúce pády tejto verzie programu"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:326 ../bin/apport-cli.py:360
#: ../bin/apport-cli.py:383
msgid "&Cancel"
msgstr "&Zrušiť"

#: ../bin/apport-cli.py:267
msgid "Problem report file:"
msgstr "Súbor hlásenia problému:"

#: ../bin/apport-cli.py:273 ../bin/apport-cli.py:278
msgid "&Confirm"
msgstr "&Potvrdiť"

#: ../bin/apport-cli.py:277
#, python-format
msgid "Error: %s"
msgstr "Chyba: %s"

#: ../bin/apport-cli.py:283 ../kde/apport-kde.py:455
msgid "Collecting problem information"
msgstr "Získavajú sa údaje o probléme"

#: ../bin/apport-cli.py:285
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Zhromaždené informácie môžu byť odoslané vývojárom na zlepšenie\n"
"aplikácie. Môže to trvať pár minút."

#: ../bin/apport-cli.py:301 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:504
msgid "Uploading problem information"
msgstr "Nahrávam informácie o probléme"

#: ../bin/apport-cli.py:303
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Zhromaždené informácie sa posielajú systému na sledovanie chýb.\n"
"Môže to trvať pár minút."

#: ../bin/apport-cli.py:359
msgid "&Done"
msgstr "&Dokončené"

#: ../bin/apport-cli.py:365
msgid "none"
msgstr "žiadny"

#: ../bin/apport-cli.py:367
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Zvolené: %s. Viaceré voľby:"

#: ../bin/apport-cli.py:384
msgid "Choices:"
msgstr "Voľby:"

#: ../bin/apport-cli.py:398
msgid "Path to file (Enter to cancel):"
msgstr "Cesta k súboru (Enter pre zrušenie):"

#: ../bin/apport-cli.py:404
msgid "File does not exist."
msgstr "Súbor neexistuje."

#: ../bin/apport-cli.py:406
msgid "This is a directory."
msgstr "Toto je adresár."

#: ../bin/apport-cli.py:412
msgid "To continue, you must visit the following URL:"
msgstr "Aby ste pokračovali, musíte navštíviť následujúce URL:"

#: ../bin/apport-cli.py:415
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Môžete teraz spustiť prehliadač, alebo skopírovať URL do prehliadača na inom "
"počítači."

#: ../bin/apport-cli.py:421
msgid "Launch a browser now"
msgstr "Okamžite spustiť prehliadač"

#: ../bin/apport-cli.py:437
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Žiadne zostávajúce chybové hlásenia. Skúste --help pre viac informácií."

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Nepridávať nové traces do hlásenia, ale vypísať ich do stdout."

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Spustiť interaktívne gdb sedenie s core dump z hlásenia (-o ignorovať; "
"neprepíše hlásenie)"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Zapísať upravené hlásenie do daného súboru namiesto pozmenenia pôvodného "
"hlásenia"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr "Odstrániť core dump z hlásenia po obnove stack trace"

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "Prepísať CoreFile hlásenia"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "Prepísať ExecutablePath hlásenia"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "Prepísať ProcMaps hlásenia"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "Znova zostaviť informácie o balíku pre hlásenie"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Vytvoriť dočasný sandbox a stiahnuť/inštalovať tam potrebné balíčky a "
"symboly pre ladenie; bez tejto voľby sa bude predpokladať, že potrebné "
"balíčky a symboly pre ladenie sú už v systéme nainštalované. Argument "
"odkazuje na koreňový adresár s konfiguráciou balíčkovacieho systému; pokiaľ "
"uvediete \"system\", tak budú použité systémové konfiguračné súbory, ale "
"potom bude možné retrasovať havárie iba pre aktuálne spustené verzie."

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""
"Hlásiť postup sťahovania/inštalácie pri inštalácii balíčkov do sandboxu"

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Pre dávkové spracovanie vložiť časové razítko pred správu do logu"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Priečinok pre rozbalené súbory. Všetky následne stiahnuté a rozbalené "
"balíčky tu budú k dispozícii."

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "Nainštalovať do sandboxu extra balík (môže byť zadaný viackrát)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Cesta k súboru s autentifikačnými informáciami pre databázu pádov. Používa "
"sa, keď sa zadáva identifikátor pádu na odoslanie retraced traces zásobníka "
"(len keď ani -g, -o a -s nie sú zadané)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Zobraziť retraced traces zásobníka a vypýtať si potvrdenie predtým ako sú "
"odoslané do databázy pádov."

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Cesta k duplicitnej sqlite databáze (predvolené: žiadna kontrola duplikátov)"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr "Nepridávať do hlásenia StacktraceSource."

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr ""

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "Pripravený na odoslanie ako prílohy? [y/n]"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "Cieľový adresár existuje a nie je prázdny."

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr ""

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr ""

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr ""
"hlásiť postup sťahovania/inštalácie pri inštalácii balíčkov do sandboxu"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Chyba: %s nie je spustiteľný. Zastavujem."

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"Toto sa stalo v priebehu minulého uspania do pamäti a zabránilo systému v "
"prebudení."

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"Toto sa stalo v priebehu minulého uspania na disk a zabránilo systému v "
"prebudení."

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Počas obnovy sa spracovanie zastavilo teste pri konci a preto sa zdá že "
"skončilo normálne."

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Váš systém sa teraz môže stať nestabilným a môže byť potrebné ho reštartovať."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Ohlásiť problém..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Oznámiť nefunkčnosť vývojárom"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Prepáčte, aplikácia %s bola neočakávane ukončená."

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Prepáčte, %s bol neočakávane ukončený."

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:213 ../kde/apport-kde.py:265
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Prepáčte, %s zaznamenal vnútornú chybu."

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:206
msgid "Send"
msgstr "Odoslať"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:660 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:329
msgid "Show Details"
msgstr "Zobraziť podrobnosti"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:257 ../kde/apport-kde.py:275
msgid "Continue"
msgstr "Pokračovať"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "Aplikácia %s neodpovedá."

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Program „%s“ neodpovedá."

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:221
#, python-format
msgid "Package: %s"
msgstr "Balík: %s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:228
msgid "Sorry, a problem occurred while installing software."
msgstr "Prepáčte, pri inštalácii softvéru sa vyskytla chyba."

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:237
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "V aplikácii %s došlo k chybe."

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:244
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Aplikácia %s bola neočakávane ukončená."

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:270
msgid "If you notice further problems, try restarting the computer."
msgstr "Ak zaznamenáte ďalšie problémy, skúste reštartovať počítač."

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:277
msgid "Ignore future problems of this type"
msgstr "Ignorovať budúce problémy tohto typu"

#: ../gtk/apport-gtk.py:664 ../kde/apport-kde.py:326
msgid "Hide Details"
msgstr "Skryť podrobnosti"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:545 ../kde/apport-kde.py:588
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Zrušiť"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "OK"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Chybové hlásenie"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Prepáčte, nastala vnútorná chyba.</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "Pamätať si toto v budúcnosti"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Ignorovať budúce problémy tejto verzie programu"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "Znovu spustiť tuto aplikáciu"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "_Preskúmať lokálne"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "Neodosielať"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Zhromažďujú sa informácie o probléme</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Zhromažďujú sa údaje pre vývojárov potrebné na opravu vami ohláseného "
"problému."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Nahrávajú sa informácie o probléme</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:506
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Zhromaždené informácie budú odoslané do systému pre sledovanie chýb, čo môže "
"chvíľu trvať."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport crash súbor"

#: ../kde/apport-kde.py:260
msgid "Leave Closed"
msgstr "Ponechať zatvorené"

#: ../kde/apport-kde.py:261 ../kde/apport-kde.py:425
msgid "Relaunch"
msgstr "Znova spustiť"

#: ../kde/apport-kde.py:354
msgid "Username:"
msgstr "Používateľské meno:"

#: ../kde/apport-kde.py:355
msgid "Password:"
msgstr "Heslo:"

#: ../kde/apport-kde.py:454
msgid "Collecting Problem Information"
msgstr "Získavajú sa informácie o probléme"

#: ../kde/apport-kde.py:457
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Zozbierané informácie môžu byť odoslané vývojárom pre zlepšenie aplikácie. "
"Môže to trvať pár minúť."

#: ../kde/apport-kde.py:503
msgid "Uploading Problem Information"
msgstr "Odosielajú sa informácie o probléme"
